client/test/index.test.js

```diff
@@ -0,0 +1,48 @@
+const createServer = require('../..')
+const Client = require('..')
```

Comment:
Any reason you chose to name it `Client` here and `Proxy` in the real thing?

Reply:
Nope. Fixed by renaming to `Client` everywhere.